features/git-town-prepend/on-feature-branch.feature

```diff
@@ -35,23 +35,3 @@ Feature: Prepending a branch to a feature branch
       | BRANCH           | PARENT     |
       | existing-feature | new-parent |
       | new-parent       | main       |
-
-
-  Scenario: Undo
-    Given I run `git town-prepend new-parent`
-    When I run `git town-prepend --undo`
-    Then it runs the commands
-        | BRANCH           | COMMAND                       |
-        | new-parent       | git add -A                    |
-        |                  | git stash                     |
-        |                  | git push origin :new-parent   |
-        |                  | git checkout main             |
-        | main             | git branch -d new-parent      |
-        |                  | git checkout existing-feature |
-        | existing-feature | git stash pop                 |
-    And I end up on the "existing-feature" branch
-    And I still have my uncommitted file
-    And I am left with my original commits
-    And Git Town is now aware of this branch hierarchy
-      | BRANCH           | PARENT |
-      | existing-feature | main   |
```

Comment:
Removed this since hack does not currently undo. We can add it back later if its worth it.